pkg/oc/admin/diagnostics/diagnostics/client/run_diagnostics_pod.go

```diff
@@ -78,22 +82,28 @@ func (d *DiagnosticPod) runDiagnosticPod(r types.DiagnosticResult) {
 	if loglevel > 2 {
 		loglevel = 2 // need to show summary at least
 	}
-	imageName := d.ImageTemplate.ExpandOrDie("deployer")
+
+	if len(d.Image) == 0 {
+		d.Image = getDefaultImage()
```

Comment:
Previously, if the user put --images=foo/${component}:${version} that would be expanded. With this change, only the default template gets expanded, so the user must fully specify the image name and version to override. That seems inconsistent and not good for automation.